backend/src/main.py

```diff
@@ -490,18 +491,13 @@ def addEmployee():
     if existing_project==None:
         return jsonify({'error':'Project with ID: {} Does not Exist !'.format(project_code)})
 
-    project_list = existing_project.split(",")
-    for i in project_list:
-        if i=="":
-            project_list.remove(i)
+    project_list=helper1(existing_project)
+    removenull(project_list)
```

Comment:
give some name for the helper1 function like "listToString"/"StringToList"
the remove null should not  be a separate function but should be a part of the helper1 function